user_stories.md

```diff
@@ -0,0 +1,11 @@
+# Пользовательские истории.
+
+1. Как пользователь, я хочу иметь возможность создавать сокращенные ссылки без регистрации на сервисе (возможно, с ограниченным временем работы ссылок), чтобы их было удобнее использовать в текстах, так как они[ccылки] будут со значительно меньшим числом символов.
+2. Как пользователь, я хочу переходить по сокращенной ссылке на страницу, которую указывал создатель этой ссылки.
+3. Как пользователь, я хочу узнать, куда ведет сокращенная ссылка, не переходя по ней явно, чтобы обезопасить себя от возможных вирусов.
```

Comment:
It is a good point about seeing the original link. Do you imagine how to implement this?
What do you mean mentioning "viruses"?

Reply:
> Do you imagine how to implement this?

Yes. It can be a request like "koro.che/{short_link}/real", which will return a real link in JSON. On the frontend, it can be a separate page between the site of link usage and the target site (site of the real link), which will show user the real link and ask them whether they like to proceed. 

> What do you mean mentioning "viruses"?

Will add it to stories.